### DISCUSSION TITLE
Bitcart project donation info

### DISCUSSION THREAD

**Discussion by Jayd603** [MrNaif2018/bitcart#380] [General]
The donation amount is too small on the front page for the amount I want to send.  Are there some direct addresses to send donations to or a way to change the send amount?   ...or I guess I can just over pay the invoice on the front page?

Thank you.

**Comment by MrNaif2018** [comment]
Hi! Yes, you can simply overpay invoice on the front page. Or in the store select many products at once. But indeed, adding custom amount to the store is on our todo list (: Thank you!

**Comment by Jayd603** [comment]
Ahh ok, sent!   I used the github link above to get to the store and just added more items.

**Reply by MrNaif2018** [reply]
Thank you very much!
